ui/app/templates/vault/cluster/access/mfa/index.hbs

```diff
@@ -20,14 +20,12 @@
       Learn more
     </DocLink>
   </p>
-  <button
+  <Hds::Button
+    @text="Configure MFA"
     type="submit"
```

Comment:
As a cleanup task, it looks like this type can be removed since this button transitions to a new route?